src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -75,9 +73,6 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             configurationProvider = extensionHost.GetService<IConfigurationProviderService>();
             configurationProvider.AssertLocalServiceIsNotNull();
 
-            errorListInfoBarController = extensionHost.GetService<IErrorListInfoBarController>();
-            errorListInfoBarController.AssertLocalServiceIsNotNull();
```

Comment:
Previously the `ActiveSolutionBoundTracker` knew about the `IErrorListInfoBarController` and called it directly when solution binding changes occurred.

This was always a hack; the `ErrorListInfoBarController` should have been listening to the events from the `ActiveSolutionBoundTracker`. I can't remember why the hack was necessary - perhaps an event-ordering problem (i.e. because we couldn't rely on the events being handled in a particular order, so the `ActiveSolutionBoundTracker` explicitly called the `ErrorListInfoBarController` after the events had been raised and handled).

Whatever the reason, it's gone now.